ONSdigital/sixteens
# Fix - Reflow - Overflow scroll with shadows

### What
Use scrollable overflows with shadows for charts so that the users aren't forced to scroll horizontally on small and medium viewports.

### How to review
1. Visit an article or bulletin with a chart
1. At small and medium viewport sizes see that they need to scroll horizontally
1. Switch to this branch and checkout babbage branch `fix/overflow-shadows`
1. See that there is now a scroll bar and shadows that appear showing that there is more content.

### Who can review
Anyone but me
